### ISSUE TITLE
Visual Page Builder broken - shortcode in Text Block - THIS IS NOT FIXED 

### ISSUE THREAD

**Issue by Dorf** [ThemeFuse/Unyson#500]
everything is appearing as a shortcode in a "Text Block" why are you not fixing this!!!

This is fucking over me and my client. How is this acceptable to you?!?!?!

![broken](https://cloud.githubusercontent.com/assets/490985/7360388/7bdff696-ed07-11e4-88d0-8dc77a109a2b.png)

**Comment by danyj** [comment]
Have you downloaded and tested the master releases? 
https://github.com/ThemeFuse/Unyson-Extensions-Approval/issues
New version is not out but the fixes are in. 
Already  broken posts cant be fixed , but the bug has been fixed.

**Comment by Dorf** [comment]
I started issue #424, when it was closed I did download and add the newest version of Unyson 2.2.8. I recreated the homepage and it broken just the same. So no the bug has not been fixed.

And, there has to be some process by which broken posts can be fixed, even if it is editing the database directly. What has to be done to fix broken posts?

**Comment by danyj** [comment]
bug was  not in Unsyon , bug was in builder 
released  yesterday 
https://github.com/ThemeFuse/Unyson-Builder-Extension/releases

Unsyon was released last 21 days ago 
https://github.com/ThemeFuse/Unyson/releases

you need to check on releases and signup to track testers, I know is bit confusing but you cant get it fixed or tested unless you track all of the repos, look here 

https://github.com/ThemeFuse/Unyson-Extensions-Approval/issues/58

I tested that bug when pending approval was out 

as for the fix on old posts I dont know , @GheorgheP might be able to help

**Comment by Dorf** [comment]
I have added the latest Page Builder and Shortcode extensions. Now any content I create in the page builder does not appear *AT ALL* on the frontend. 

ex. http://giannaandcompany.com/test-if-i-show/ 

![broken1](https://cloud.githubusercontent.com/assets/490985/7376514/74bbbcd8-ed9d-11e4-9684-06869a08c36c.png)

![broken2](https://cloud.githubusercontent.com/assets/490985/7376519/7a12eb70-ed9d-11e4-8c46-7074be5d41d0.png)

**Comment by 2Fwebd** [comment]
No news about this issue ? Clients have the same issue here... After testing, it seems to happen only for Pages with a *Page Template* set. I hope that can help, let me know if you need more details ;)
2F

**Comment by moldcraft** [comment]
@2Fwebd can you give me an FTP where this happens so I can debug it?

Yesterday we released an update for PageBuilder extension, this bug still happens with the latest extensions (check the updates page)

**Comment by 2Fwebd** [comment]
@moldcraft sent a minute ago ;)

2F

**Comment by moldcraft** [comment]
@2Fwebd Thank you

Everything worked fine except the Preview:

* I made some changes in builder and pressed Preview
* The preview didn't displayed the changes, but displayed what was saved in db
* Then I pressed the Update post button and all elements from the builder were wiped out (but I was able to make a recover from revisions)

Anyway, I fixed the preview in the latest release.

I checked the updates page and there was [updates for extensions](https://static.md/a3a008d4ac38edde96ce2e262f7ce35b.png). I [updated the extension](https://static.md/1c0a9d29ec1946fec98a3acbda06015f.png). Now the preview works fine.
